packages/core/src/components/button/button-vars.scss

```diff
@@ -27,25 +27,24 @@
   //Secondary
   --tds-btn-secondary-background: transparent;
   --tds-btn-secondary-color: var(--tds-black);
-  --tds-btn-secondary-border-color: rgb(0 0 0 / 48%);
+  --tds-btn-secondary-border-color: var(--tds-black-48);
   --tds-btn-secondary-background-hover: var(--tds-blue-500);
   --tds-btn-secondary-color-hover: var(--tds-white);
   --tds-btn-secondary-border-color-hover: var(--tds-blue-500);
   --tds-btn-secondary-background-active: transparent;
   --tds-btn-secondary-color-active: var(--tds-black);
-  --tds-btn-secondary-border-color-active: rgb(0 0 0 / 87%);
+  --tds-btn-secondary-border-color-active: var(--tds-black-87);
   --tds-btn-secondary-background-focus: transparent;
   --tds-btn-secondary-color-focus: var(--tds-black);
   --tds-btn-secondary-border-color-focus: var(--tds-blue-400);
-  --tds-btn-secondary-outline-color: rgb(0 0 0 / 38%);
+  --tds-btn-secondary-outline-color: var(--tds-black-38);
   --tds-btn-secondary-outline-color-focus: var(--tds-blue-400);
   --tds-btn-secondary-background-disabled: transparent;
-  --tds-btn-secondary-color-disabled: rgb(0 0 0 / 38%);
-  --tds-btn-secondary-border-color-disabled: rgb(0 0 0 / 38%);
+  --tds-btn-secondary-color-disabled: var(--tds-black-38);
+  --tds-btn-secondary-border-color-disabled: var(--tds-black-38);
   --tds-btn-icon-secondary-color-focus: var(--tds-black);
   --tds-btn-icon-secondary-fill-focus: var(--tds-black);
-  --tds-btn-icon-secondary-fill-active: var(--tds-black);
-  --tds-btn-icon-secondary-fill-active: var(--tds-black);
+  --tds-btn-icon-secondary-fill-active: var(--tds-grey-958);
 
```

Comment:
On the Primary button, in dark mode and in active (pressed) state, there is an outline with Blue 700 while the rest of the Button is Blue 300. In Figma there is no outline at all, only the Blue 300 background color. Should this be changed?

In Figma
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/d8972e59-e8af-43b6-b8a4-e777a6666b56)


In the PR:
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/de9bbd30-45e3-44ae-ba4c-9918f5c6f4c5)

Reply:
Same as one before, it is too many issues under packages/core/dist/collection/components/button/button.css and this is just one of them. Issues needs its own ticket and time to fix.